contribute-to-tidb/write-document.md

```diff
@@ -1,3 +1,229 @@
 # Write Document
 
 Good documentation is crucial for any kind of software. This is especially true for sophisticated software systems such as distributed database like TiDB. The TiDB community aims to provide concise, precise, and complete documentation and welcomes any contribution to improve TiDB's documentation.
+
+## What you can contribute
+
+You can start from any one of the following items to help improve TiDB Docs at the PingCAP website ([en](https://docs.pingcap.com/tidb/stable) and [zh](https://docs.pingcap.com/zh/tidb/stable)):
+
+- Fix typos or format (punctuation, space, indentation, code block, etc.)
+- Fix or update inappropriate or outdated descriptions
+- Add missing content (sentence, paragraph, or a new document)
+- Translate docs changes from English to Chinese, or from Chinese to English
+- Submit, reply to, and resolve [docs issues](https://github.com/pingcap/docs/issues)
+- (Advanced) Review Pull Requests created by others
+
+## Before you contribute
+
+Before you contribute, please take a quick look at some general information about TiDB documentation maintenance. This can help you to become a contributor soon.
+
+### Get familiar with style
+
+- [Commit Message Style](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md#how-to-write-a-good-commit-message)
+- [Pull Request Title Style](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md#pull-request-title-style)
+- [Markdown Rules](/resources/markdownlint-rules.md)
+- [Code Comment Style](https://github.com/pingcap/community/blob/master/contributors/code-comment-style.md)
+- Diagram Style: [Figma Quick Start Guide](https://github.com/pingcap/community/blob/master/contributors/figma-quick-start-guide.md)
+
+    To keep a consistent style for diagrams, we recommend using [Figma](https://www.figma.com/) to draw or design diagrams. If you need to draw a diagram, refer to the guide and use shapes or colors provided in the template.
```

Comment:
This paragraph seems bad formatted. Is it a note or comment?

Reply:
It is a common format used for the paragraph that belongs to a list item. It is commonly used in TiDB documentation.